src/lib/courses.ts

```diff
@@ -34,9 +34,9 @@ const courseQuery = groq`
     'avatar_url': person->image.url,
     'slug': person->slug.current
   },
-	'dependencies': softwareLibraries[]{
+	'dependencies': topicTaggings[]{
     version,
-    ...library->{
+    ...topic->{
```

Comment:
This is the only `groq` query referencing `softwareLibraries`. Updating it to match the internal schema.

This query ends up calling the resulting array of data `dependencies` which is a little but of a misnomer for things like `a11y`, but it works.

Reply:
I think we need a set of `conceptTags` or similar from a taxonomy perspective.